lib/hammer_cli_katello/repository.rb

```diff
@@ -35,6 +35,8 @@ class InfoCommand < HammerCLIKatello::InfoCommand
         field :full_path, _("Published At")
         field :docker_upstream_name, _("Upstream Repository Name"),
               Fields::Field, :hide_blank => true
+        field :container_repository_name, _("Container Repository Name"),
```

Comment:
In your API PR, it looks like you have :container_image_name still:

https://github.com/parthaa/katello/commit/f5d230b0b5cf27f5f03b21fdac59bda174a4a796#diff-30fda49df0f72dc1d194c8e6ec02b4a0R9